tests/utils/shippable/units.sh

```diff
@@ -22,6 +22,11 @@ esac
 
 ansible-test env --timeout "${timeout}" --color -v
 
+if [ "$2" == "2.10" ]; then
+    sed -i -E 's/^python-gitlab($| .*)/python-gitlab < 2.10.1 ; python_version >= '"'"'3.6'"'"'/g' tests/unit/requirements.txt
```

Comment:
It might be easier to read it as:
```suggestion
    sed -i -E 's/^python-gitlab($| .*)/python-gitlab < 2.10.1 ; python_version >= '\'3.6\''/g' tests/unit/requirements.txt
```